cirq-core/cirq/study/sweeps.py

```diff
@@ -196,6 +197,9 @@ def param_tuples(self) -> Iterator[Params]:
     def __repr__(self) -> str:
         return 'cirq.UnitSweep'
 
+    def _json_dict_(self) -> Dict[str, Any]:
```

Comment:
Based on the name, `_Unit` is a private class. Do we want it to be serializable?

If so, I think this needs a `_from_json_dict_` classmethod.

Reply:
SGTM. This can merge once the tests are sorted out.